_build/templates/default/package.json

```diff
@@ -1,29 +1,40 @@
 {
-  "name": "revolution-theme-default",
-  "title": "MODX Revolution",
-  "version": "3.0.0",
-  "license": "GPL-2.0+",
-  "repository": "https://github.com/modxcms/revolution",
-  "devDependencies": {
-    "bourbon": "^5.1.0",
-    "bourbon-neat": "^3.0.0",
-    "grunt": "^1.4.1",
-    "grunt-autoprefixer": "^3.0.4",
-    "grunt-contrib-concat": "^1.0.1",
-    "grunt-contrib-copy": "^1.0.0",
-    "grunt-contrib-csslint": "^2.0.0",
-    "grunt-contrib-cssmin": "^4.0.0",
-    "grunt-contrib-watch": "^1.1.0",
-    "grunt-dart-sass": "^2.0.1",
-    "grunt-notify": "^0.4.5",
-    "grunt-terser": "^2.0.0",
-    "node": "^14.15.0",
-    "normalize-scss": "^7.0.1"
-  },
-  "dependencies": {
-    "@fortawesome/fontawesome-free": "^5.15.4"
-  },
-  "scripts": {
-    "build": "grunt build"
-  }
+    "name": "revolution-theme-default",
+    "title": "MODX Revolution",
+    "version": "3.0.0",
+    "license": "GPL-2.0+",
+    "repository": "https://github.com/modxcms/revolution",
+    "devDependencies": {
+        "@lodder/grunt-postcss": "^3.0.1",
+        "autoprefixer": "^10.3.3",
+        "bourbon": "^5.1.0",
+        "bourbon-neat": "^3.0.0",
+        "cssnano": "^5.0.8",
+        "grunt": "^1.4.1",
+        "grunt-contrib-concat": "^1.0.1",
+        "grunt-contrib-copy": "^1.0.0",
+        "grunt-contrib-watch": "^1.1.0",
+        "grunt-dart-sass": "^2.0.1",
+        "grunt-notify": "^0.4.5",
+        "grunt-terser": "^2.0.0",
+        "node": "^14.15.0",
+        "normalize-scss": "^7.0.1",
+        "postcss": "^8.3.6"
+    },
+    "dependencies": {
+        "@fortawesome/fontawesome-free": "^5.15.4"
+    },
+    "scripts": {
+        "build": "grunt build"
+    },
+    "browserslist": [
+        "Chrome >= 64",
+        "Firefox >= 78",
+        "Safari >= 12",
+        "Edge >= 79",
+        "iOS >= 11",
+        "ChromeAndroid >= 92",
+        "last 2 Android versions",
+        "not dead"
+    ]
```

Comment:
Can you share why you chose these browsers and versions specifically?

Reply:
Yes - did a little looking on caniuse and chose versions that supported at least ES6 (fully) and up to ES8 (mostly), taking into account the worldwide usage percentages. Originally the setting in the gruntfile (in the demoted autoprefixer's config) was 'last 2 versions', but given how quickly some of the browser vendors change even major versions, it seemed that could become too edgy.

Note, in general, I'm not going to be in the habit of changing my PRs that are already under review (unless requested), but it was bugging me as I continued to test the workflow that grunt-autoprefixer kept raising warnings about vulnerabilities in the version of diff that it uses ... motivated me to swap in the more actively maintained and modern postCSS.